.github/workflows/ci.yml

```diff
@@ -2,6 +2,10 @@ name: 'Push CI'
 
 on:
   push:
+    tags-ignore:
+      - "v**"
+      - 'buffer-v**'
+      - 'templates-api-v**'
```

Comment:
Testy sa sputaju vramci `CLI/templates/buffer` release, nie je potrebne ich spustat este samostatne, ked to otagujeme.